Client/src/App.jsx

```
@@ -39,6 +39,14 @@ import { useDispatch } from "react-redux";
 import { getAppSettings, updateAppSettings } from "./Features/Settings/settingsSlice";
 import { logger } from "./Utils/Logger"; // Import the logger
 import { networkService } from "./main";
+import { setMode } from "./Features/UI/uiSlice";
+
+const getPreferredTheme = () => {
+	if(window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)